src/taipy/config/common/_template_handler.py

```diff
@@ -72,33 +72,33 @@ def _replace_template(cls, template, type, required, default):
     def _to_bool(val: str) -> bool:
         possible_values = ["true", "false"]
         if str.lower(val) not in possible_values:
-            raise InconsistentEnvVariableError()
+            raise InconsistentEnvVariableError("{val} is not a boolean.")
```

Comment:
'boolean' is not a common adjective. This needs a capital letter: -> 'Boolean'.
(was invented by George Boole, hence the name)